src/test/java/core/basesyntax/strategy/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.storage.Storage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private BalanceOperationHandler balanceOperationHandler;
+
+    @BeforeEach
+    void setUp() {
+        balanceOperationHandler = new BalanceOperationHandler();
+        Storage.fruits.clear();
```

Comment:
the same, you can clear storage only after each test  (check it in other classes)